substrait/substrait-spark/pom.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
```

Comment:
@zhixingheyi-tian  did you mean this?

Reply:
I mean put into  here, https://github.com/oap-project/gluten/blob/9ed615443cb420d0682f07e7b7e70bfa00228569/pom.xml#L229

Reply:
gluten-core depends on this module, no need to put it in profiles